packages/multicast_dns/pubspec.yaml

```diff
@@ -0,0 +1,16 @@
+name: multicast_dns
+description: Dart package for mDNS queries (e.g. Bonjour, Avahi).
+author: Flutter Team <flutter-dev@googlegroups.com>
+homepage: https://github.com/flutter/packages/tree/master/packages/mdns
+version: 0.1.0
+
+dependencies:
+  args: ^1.5.0
+  collection: ^1.14.11
```

Comment:
you don't seem to actually depend on package:collection

Reply:
lookup_resolver.dart uses `LinkedListEntry`.

Reply:
If that's a problem for Flutter, it'd be trivial for this to roll its own (or potentially use a different data structure).

Reply:
Ah - I don't need package collection for that though, just dart:collection.